### PR TITLE
FOR-2714: Fix Webhook Action authentication part autocompleting

### PR DESCRIPTION
**Pull request by AlehKatsiubaFormIO**

### CODE REVIEW

**Comment by AlehKatsiubaFormIO** [comment]
Is related to https://github.com/formio/formio-server/pull/628

File: src/components/_classes/input/Input.js [hunk 1]
```diff
@@ -37,6 +37,10 @@ export default class Input extends Multivalue {
       attr.disabled = 'disabled';
     }
 
+    if (this.component.autocomplete) {
+      attr.autocomplete = this.t(this.component.autocomplete);
```
[the remark this answers is not in the document]
Comment (AlehKatsiubaFormIO) [reply src/components/_classes/input/Input.js]: Done.

File: src/components/textfield/editForm/TextField.edit.display.js [hunk 2]
```diff
@@ -127,6 +127,15 @@ export default [
     key: 'suffix',
     label: 'Suffix'
   },
+  {
+    weight: 700,
```
[the remark this answers is not in the document]
Comment (AlehKatsiubaFormIO) [reply src/components/textfield/editForm/TextField.edit.display.js:131]: Autocomplete attribute can have a complex value, for example to group addresses. [Here](https://developers.google.com/web/updates/2015/06/checkout-faster-with-autofill#address) an example. If we use Select dropdown we will lose this option.

[the remark this answers is not in the document]
Comment (travist) [reply src/components/textfield/editForm/TextField.edit.display.js:131]: Ok, then looks good.

**Pull request merged**